ui/app/assets/javascripts/fusor_ui/application_no_tl.js

```diff
@@ -0,0 +1,624 @@
+// copied from https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/application.js
+// and disabling turbolinks
+//= require jquery
+// omitting Turbolinks -- require jquery.turbolinks
```

Comment:
removed the offending turbolinks js files https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/application.js#L2-L3
but the rest is the same